assets/js/base/components/payment-methods/style.scss

```diff
@@ -1,25 +1,58 @@
+$border-width: 1px;
+
 .wc-block-components-express-checkout {
 	margin: auto;
-	border: 2px solid $black;
-	border-radius: 5px;
-	padding: 8px;
 	position: relative;
 
-	.wc-block-components-express-checkout__title {
-		background-color: $white;
-		padding-left: $gap-small;
-		padding-right: $gap-small;
-		margin-left: $gap-small;
-		display: inline-block;
-		vertical-align: middle;
-		transform: translateY(-50%);
+	.wc-block-components-express-checkout__title-container {
+		display: flex;
+		flex-direction: row;
+		left: 0;
 		position: absolute;
+		right: 0;
 		top: 0;
-		margin-top: -2px; // Same as container border width.
+		vertical-align: middle;
+
+		&::before {
+			border: $border-width solid transparent;
+			border-left-color: currentColor;
+			border-top-color: currentColor;
+			border-radius: 5px 0 0 0;
+			content: "";
+			display: block;
+			margin-right: $gap-small;
+			pointer-events: none;
+			width: #{$gap-larger - $gap-small - $border-width * 2};
+		}
+
+		&::after {
+			border: $border-width solid transparent;
```

Comment:
Are these pseudo elements used to create the "padding" around the title (i.e. overlay the other express payment border? Might be worth a little comment explaining the intention/approach, or maybe package as a mixin (or class) if that's practical.

Reply:
Good point. Added a comment: 2b6f524a1f7d8a0fcc67b4fc5f638de98a8271ac. I don't think we need to create a mixin for it yet considering we are only using it here. We can extract it in the future if we see there are other instances where we use it.

Reply:
Thanks for the visual explanation below, ingenious fix!